developer-docs-site/docs/concepts/resources.md

```diff
@@ -47,22 +47,18 @@ module 0x1234::coin {
 }
 ```
 
-In this example, `0x1234` is the address, `coin` is the module, `Coin` is a struct that can be stored as a resource, and `SomeCoin` is a struct that is unlikely to ever be represented as an object. The use of the phantom type allows for there to exist many distinct types of `CoinStore` resources with different `CoinType` parameters.
+In this example, `0x1234` is the address, `coin` is the module, `Coin` is a struct that can be stored as a resource, and `SomeCoin` is a struct that is unlikely to ever be represented as an instance. The use of the phantom type allows for there to exist many distinct types of `CoinStore` resources with different `CoinType` parameters.
 
-## Dual ownership of objects, including resources
+## Permissions of Instances including Resources
```

Comment:
If the section is about resources, should we talk about resources only then, I am wondering? Here we mention `instance within a resource may be accessed and even removed from the resource` but in general this is a useless information, we just say that resource fields can be accessed?

Reply:
When learning to develop in Move, I think it is important to state
1. key is required for top level in global storage, this is a resource
2. store is required for storage as field within a resource or other stored field

While intuitive, it isn't an immediate conclusion, so writing it out helps the reader get that clarity more quickly.